rfc002-procope-blog.md

```diff
@@ -0,0 +1,208 @@
+
+This is largely contrary to other open-source stances, but is important for three reasons:
+
+1. We want to allow contributors a space to incubate their ideas, which means in some cases a space to develop IP they want to work with later—such as in a talk or book down the road. And we want our contributors to feel good about having added to GDP's body of knowledge, rather than regret "giving up a good idea."
+2. There is an unfortunate history of privileged classes taking work created by marginalized people, and profiting off of that work without recognition of or compensation to those people. Because of the individual nature of each post and video, we should strive to not be a part of that history.
+3. Such media doesn't need to be replicated and altered in the way code bases are. Code bases and similar need legal declarations to protect those using them. The replication and alteration process is a personally interpreted one, and it doesn't necessitate a legal declaration.
+
+### Publication cadence
+
+We want to soft-launch our blog content in July 2021, and video content launching in Q4-2021.
+
+Initially, blog posts would come out once a week. We would want to move to a twice-weekly cadence by January 2022.
+
+Videos would ideally come out once every two weeks, but that may not be sustainable. Our goal within the first six months of video content is to have a sustainable model every two weeks, and perhaps to eventually every week if there's enough interest in continued production. (The labor and barrier to skill entry involved with video production means we shouldn't push this hard and overcommit.)
+
+We want to build up to a six-item queue of content. That may be harder for videos, and in contrast for blog posts we may have a large "slush pile" to work through.
+
+## Open Questions (non-blocking)
+
+* Regarding individual content elements: Is there a way we can utilize issues in github, or is by the nature of our publication model something that isn't in github at all? Likely the latter, in which case we could use a form + sheet.
+
+## Reference Links
+
+* [Procope draft responsibility model, brainstormed late April 2021](https://jamboard.google.com/d/1iyfr4A9GcdRbvnkrokfg6YGXTBhIgmeA55UTBvtN_-o/viewer)
+* [Procope's Google Drive folder](https://drive.google.com/drive/folders/1UCIwyvmOa4Gh5nBmXu1GckrI7mfeT2AW?usp=sharing]) (owned by Ryan's personal account)
+* [Procope's meeting notes](https://docs.google.com/document/d/18T2dku9WMRLJTq1rcnFwuUeA1i-thVLOtVqhX2SOxIY/edit?usp=sharing) (last meeting in late April; meeting cadence is TBD)
+* [Content style guide](https://docs.google.com/document/d/1St73OBFhbQT4hNKp-cayTjDfjOltZSMP7X2Z4QBzgQo/edit) (in notes form, not yet following a template)
+* [GDP meeting notes](https://docs.google.com/document/d/13xr5PLMMqm6Rz_yXzxl5WMOwY9H6liB1abU_i0-ws60/edit#) (where various Procope discussions happen at a larger-org level)
+
+## Initial Implementation Checklist
```

Comment:
Notes from the syndication comments show we should add a beginning Policies & Procedures doc here. Procedures include mechanics of posting and of editing/updating. Policies include minor and major retraction policies (obviously untested at first).

Reply:
suggestion (non-blocking): weekly schedule is very ambitious! I would maybe suggest every other week to start, or even monthly, and increase as submissions increase?

Reply:
I was in fact thinking this a couple nights ago as I was trying to fall asleep, so yeah, I'm happy to start slow rather than even that aggressive :)